Sources/Rosetta/PlayMode/CardSets/ScholomanceCardsGen.cpp

```diff
@@ -1826,6 +1830,14 @@ void ScholomanceCardsGen::AddNeutral(std::map<std::string, CardDef>& cards)
     // Text: <b>Spellburst:</b> Add 2 random spells
     //       from your class to your hand.
     // --------------------------------------------------------
+    power.ClearData();
+    power.AddSpellburstTask(std::make_shared<RandomCardTask>(
+        CardType::SPELL, CardClass::PLAYER_CLASS));
+    power.AddSpellburstTask(std::make_shared<AddStackToTask>(EntityType::HAND));
+    power.AddSpellburstTask(std::make_shared<RandomCardTask>(
+        CardType::SPELL, CardClass::PLAYER_CLASS));
+    power.AddSpellburstTask(std::make_shared<AddStackToTask>(EntityType::HAND));
```

Comment:
You can refactor this code using `EnqueueTask`, but it's not a problem.
I'll change code later.